### DISCUSSION TITLE
Why is there a model and an entity?

### DISCUSSION THREAD

**Discussion by alper** [SeaQL/sea-orm#482] [Q&A] [answered]
I don't get why this duplication is there and it's confusing which to use in any given situation.

**Comment by billy1624** [comment, marked as answer]
Hey @alper, basically `Model` represents a row of record in database. Whereas `Entity` glue all the specifications (columns info, primary key info, model info, etc.) of a database table using a unit struct.

REF: https://www.sea-ql.org/SeaORM/docs/generate-entity/entity-structure

**Comment by alper** [comment]
Here, now I ended up with this: `<foursquare_gmaps_mappings::Entity as Trait>::post::ActiveModel`

Totally unclear what to import where and how.

**Reply by billy1624** [reply]
Why not import `post::ActiveModel` into the scope then use it directly?

**Reply by alper** [reply]
Now I have indeed: `entity:: foursquare_gmaps_mappings::ActiveModel` that I'm using.

**Reply by billy1624** [reply]
Nice!
